tsconfig.json

```diff
@@ -23,11 +22,14 @@
     "suppressImplicitAnyIndexErrors": true,
     "noUnusedLocals": true,
     "resolveJsonModule": true,
-    "typeRoots": []
+    "typeRoots": ["node_modules/@trustwallet/types"]
   },
-  "include": [],
+  "include": ["./node_modules/@trustwallet"],
   "references": [
     { "path": "./packages/ui" },
-    { "path": "./packages/walletconnect" }
+    { "path": "./packages/walletconnect" },
+    { "path": "./packages/provider" },
+    { "path": "./packages/types" },
+    { "path": "./packages/eslint-config" }
```

Comment:
Is there a way to specify as `./packages/**` ?

Reply:
I'd rather leave it like this now. Can change it later. For example we don't want the `./packages/examples` in this